### PR TITLE
(back port) Miscellaneous New and Noteworthy fixes for 4.27

### PR DESCRIPTION
**Pull request by gireeshpunathil**
* Miscellaneous New and Noteworthy fixes for 4.27

-fix incorrect versioning
-load missing images (End user reported)
-fix incorrect html tagging

---------

/cc @lshanmug

### CODE REVIEW

File: bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF [hunk 1]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.jdt.doc.user; singleton:=true
-Bundle-Version: 3.15.1700.qualifier
+Bundle-Version: 3.15.1701.qualifier
```
Comment (lshanmug) [inline bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF:5]: jdt.doc.user has no changes, is version bump required?

File: bundles/org.eclipse.platform.doc.isv/META-INF/MANIFEST.MF [hunk 2]
```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.platform.doc.isv; singleton:=true
-Bundle-Version: 4.27.0.qualifier
+Bundle-Version: 4.27.1.qualifier
```
Comment (lshanmug) [inline bundles/org.eclipse.platform.doc.isv/META-INF/MANIFEST.MF:5]: corresponding pom update required

File: bundles/org.eclipse.platform.doc.user/whatsNew/platform_whatsnew.html [hunk 3]
```diff
@@ -21,10 +21,10 @@
   ul {padding-left: 13px;}
   /*]]>*/
   </style>
-  <title>Eclipse Platform What's New in 4.25</title>
+  <title>Eclipse Platform What's New in 4.27</title>
```
Comment (lshanmug) [inline bundles/org.eclipse.platform.doc.user/whatsNew/platform_whatsnew.html:24]: pom and manifest version update required

**Review by lshanmug — CHANGES_REQUESTED** [review]

File: bundles/org.eclipse.platform.doc.user/whatsNew/platform_whatsnew.html (hunk 3 above)
Comment (gireeshpunathil) [reply bundles/org.eclipse.platform.doc.user/whatsNew/platform_whatsnew.html:24]: https://github.com/eclipse-platform/eclipse.platform.common/blob/184b0084227904dd2ddd5d71d13afd298358f875/bundles/org.eclipse.jdt.doc.isv/pom.xml#L20-L23

it does not have a version section; looks like it inherits the parent's one?

File: bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF (hunk 1 above)
Comment (gireeshpunathil) [reply bundles/org.eclipse.jdt.doc.user/META-INF/MANIFEST.MF]: fixed, sorry for the goof-up!

File: bundles/org.eclipse.platform.doc.isv/META-INF/MANIFEST.MF (hunk 2 above)
Comment (gireeshpunathil) [reply bundles/org.eclipse.platform.doc.isv/META-INF/MANIFEST.MF:5]: https://github.com/eclipse-platform/eclipse.platform.common/blob/8c24111566f2be2018deaec632fdb29c05683c2d/bundles/org.eclipse.platform.doc.isv/pom.xml#L20-L23

it does not have a version section; looks like it inherits the parent's one?

Comment (lshanmug) [reply bundles/org.eclipse.platform.doc.isv/META-INF/MANIFEST.MF:5]: line17 has the version

**Review by lshanmug — APPROVED** [review]

**Pull request merged**